src/components/UserLogin.vue

```diff
@@ -18,14 +18,15 @@
       </div>
 
       <div class="form-actions">
-        <input type="submit" value="Login" />
+        <p><input type="submit" value="Login" /> &ensp; Don't have an account yet? <router-link :to="{path: '/registration'}">Register</router-link></p>
       </div>
     </form>
   </div>
 </template>
 
 <script>
   import axios from 'axios';
+  import router from "../router";
```

Comment:
We don't need the router import in this component, do we?
We can already access the Application router component via `this.$route` if needed.